src/deploy/function_account_birth_date_set.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+
+CREATE FUNCTION vibetype.account_birth_date_set(
+  birth_date DATE
+) RETURNS VOID AS $$
+BEGIN
+  UPDATE vibetype_private.account
+    SET birth_date = account_birth_date_set.birth_date
+    WHERE id = vibetype.invoker_account_id();
+
+  IF NOT FOUND THEN
+    RAISE EXCEPTION 'No account found with the specified ID'
+      USING ERRCODE = 'P0002';
+  END IF;
+END;
+$$ LANGUAGE PLPGSQL STRICT VOLATILE SECURITY DEFINER;
+
+COMMENT ON FUNCTION vibetype.account_birth_date_set(DATE) IS E'Sets the birth date for the invoker''s account.\n\nError codes:\n- **P0002** when no record was updated';
```

Comment:
@sthelemann this comment documents an error code returned by the function and shows up like this in Postgraphile's schema explorer:

![image](https://github.com/user-attachments/assets/ac46fea3-1a33-4902-a962-1d026b129f67)